src/mixins/raster-layer-point-mixin.js

```diff
@@ -199,6 +225,20 @@ function getTransforms(
     })
   }
 
+  const postFilter = postFilters ? postFilters[0] : null // may change to map when we have more than one postFilter
+  if (postFilter && isValidPostFilter(postFilter)) {
```

Comment:
So if we have an invalid postFilter, we just silently ignore it? Is that ever expected to happen, in normal operation?

Reply:
it is basically checks relation between operator and min or max values and the the value type. If it is not satisfies the minimal requirement, yes we ignore the filter get applied.